notebooks/official/pipelines/google_cloud_pipeline_components_bqml_text.ipynb

```diff
@@ -0,0 +1,1460 @@
+{
```

Comment:
<p>is it necessary to pin versions for?: google-cloud-aiplatform==1.10.0 google_cloud_pipeline_components==1.0.0&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I was thinking about reproducibility Andrew. But I can unpin if you want</p>

Reply:
done